platform/linux-generic/pktio/dpdk.c

```diff
@@ -339,12 +340,18 @@ static struct rte_mempool *mbuf_pool_create(const char *name,
 		goto fail;
 	}
 
+	/* Release notes v19.11: Changed the mempool allocation behaviour
+	 * so that objects no longer cross pages by default */
+#if RTE_VERSION >= RTE_VERSION_NUM(19, 11, 0, 0)
+	flags |= MEMPOOL_F_NO_IOVA_CONTIG;
+#endif
+
 	/* Skipped buffers have to be taken into account to populate pool
 	 * properly. */
 	mp = rte_mempool_create_empty(name, num + pool_entry->skipped_blocks,
 				      elt_size, cache_size(num),
 				      sizeof(struct rte_pktmbuf_pool_private),
-				      rte_socket_id(), 0);
+				      rte_socket_id(), flags);
```

Comment:
According to DPDK API the same flags should also be used for `rte_mempool_calc_obj_size()`.

Reply:
fixed in v2